src/main/content/_assets/css/blog.scss

```diff
@@ -217,7 +217,13 @@ html {
     font-size: 16px;
     color: #C9D6FE;
     & > a {
-        color: #C9D6FE
+        color: #C9D6FE;
+        @media (max-width: 767.98px) {
+            color: #5E6B8D;
+            &:hover {
+                color: #3F4659 !important;
```

Comment:
Is `!important` still needed now that this styling as been moved higher up in the `.css` file to get more priority?